src/Input.cpp

```diff
@@ -7,22 +7,22 @@ void init_input()
   SDL_SetRelativeMouseMode(SDL_TRUE);
 }
 
-void handle_keyboard_input(Camera &camera, bool &loop)
+void handle_keyboard_input(Camera* camera, bool &loop)
```

Comment:
Still not all on board on this "pointer is better" argument since one can pass a pointer wo changing anything - but I'm fine with the change